elide-datastore/elide-datastore-aggregation/src/test/resources/configs/models/tables/SalesView.hjson

```diff
@@ -165,6 +163,16 @@
             sql: PARSEDATETIME(FORMATDATETIME({{}}, 'yyyy'), 'yyyy')
           }
         }
+        {
+          name: deliveryDefault
+          type: TIME
+          // Logical Column Reference in same table, which references another Logical Column in referred table, which references another Logical column in referred table, which references another Logical column in referred table, which references Physical column in referred table
+          definition: '{{delivery.time}}'
+          readAccess: Principal is guest user
+          grain:
```

Comment:
What happens if we remove the grain definition here?  I would expect the developer to not put it here empty like this.

Reply:
Let's remove the grain here.